Update Stookwijzer integration markdown (#30037)
home-assistant/home-assistant.github.io: source/_integrations/stookwijzer.markdown

ha_integration_type: service
---

> The Stookwijzer integration queries the [Atlas Leefomgeving Stookwijzer](https://www.atlasleefomgeving.nl/stookwijzer) API for air quality index and windspeed. Combining these values, Stookwijzer advises people not to burn pallets or wood, or to use the barbecue. This can prevent health problems for people in the area.

Reviewer: ```suggestion
The Stookwijzer integration queries the [Atlas Leefomgeving Stookwijzer](https://www.atlasleefomgeving.nl/stookwijzer) API for air quality index and windspeed. Based on these values, Stookwijzer advises people not to burn pallets or wood, or not to use the barbecue. This can help prevent health issues for people in the area.
```

> The Stookwijzer can provide three different advices:

Reviewer: ```suggestion
The Stookwijzer integration can provide three different levels of recommendations:
```

> - Code Yellow: Please note: heating causes nuisance and air pollution.

Reviewer: ```suggestion
- Code Yellow: Please note: burning wood causes discomfort and air pollution.
```
Is there a code green? What happens if all is well? The system shows yellow? Or nothing at all?

> - Code Orange: It is better not to burn wood now.

Reviewer: ```suggestion
- Code Orange: It is recommended not to burn wood now.
```

> - Code Red: Do not burn wood.

Reviewer: ```suggestion
- Code Red: Do not burn wood now.
```
